website/docs/docs/third-party-react-native-vector-icons.mdx

```diff
@@ -0,0 +1,42 @@
+---
+id: third-party-react-native-vector-icons
+title: react-native-vector-icons
+sidebar_label: react-native-vector-icons
+---
+
+react-native-vector-icons is a very popular library for using SVG icons in react-native, which keeps large static images
+out of your app while providing crisp icons. Using this library with React Native Navigation is not difficult, but does take
+a few extra steps..
+
+## Using SVG icons in BottomTabs or TopBarButtons
+
+In the example below we will detail a simple app that pre-loads several icons from react-native-vector-icons to be used in our navigation. The example assumes
+you have worked through the [installation](https://github.com/oblador/react-native-vector-icons#installation) and optional steps to support [custom fonts](https://github.com/oblador/react-native-vector-icons#custom-fonts)
+according to the react-native-vector-icons project.
+
+### Step 1 - Choose icons and load image source.
+
+The core issue here is that React Native Navigation wants 'images' for its icons while react-native-vector-icon typically works as a component. However,
+there is a way around this, by using the `getImageSource` method exposed by react-native-vector-icons.
```

Comment:
There seems to also be `getImageSourceSync`. Why not use that instead?

Reply:
Yeah -- not totally sure (I wrote my code several years ago, so possible the API was extended, or maybe I just didn't use it before).

Let me try out the sync option and see how that works.

Reply:
`getImageSource` returns a promise, you have to either await it or use `getImageSourceSync`

Reply:
Okay, switched the example to use `getImageSourceSync` instead, which seems to work.